app/api/server/v1/inquiries.js

```diff
@@ -0,0 +1,37 @@
+import { Meteor } from 'meteor/meteor';
```

Comment:
Please, move your code to the Livechat API folder:
https://github.com/RocketChat/Rocket.Chat/tree/develop/app/livechat/imports/server/rest